src/Commands/CommandFactory.cs

```diff
@@ -409,6 +410,33 @@ private void RegisterRedisCommands()
         database.AddCommand("list", new Redis.ManagedRedis.DatabaseListCommand(GetLogger<Redis.ManagedRedis.DatabaseListCommand>()));
     }
 
+    private void RegisterLoadTestingCommands()
+    {
+        var service = new CommandGroup(
+            "loadtesting",
+            "Load Testing operations - Commands for managing Azure Load Testing resources.");
```

Comment:
are there any other key words that should be added here to help the Agent choose this tool over others?  The tool description is the SEO for tool selection.


See https://modelcontextprotocol.io/docs/concepts/tools#best-practices where they recommend adding examples and more verbose tool descriptions.

Reply:
@jeffreyrichter can you comment here once your tool is checked-in so the azure load test team can test their tool descriptions?